cmd/podman/pod_ps.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	STOPPED      = "Stopped"
```

Comment:
Nit: I'd love to use `define.(ContainerStatus).String()` for ps but I assume that's for Docker compat? Couldn't we add another `define.(ContainerStatus).PSString()?

Reply:
is that applicable to this PR?

Reply:
Somehow yes, as STOPPED isn't used anywhere and the linter was complaining about it. It's no blocker, and if desired, we can tackle that later when grepping for `nolint`.